src/pycontw2016/settings/base.py

```diff
@@ -329,9 +330,8 @@ def node_bin(name):
 )
 
 EVENTS_DAY_NAMES = collections.OrderedDict([
-    (datetime.date(2019, 9, 20), _('Day 1')),
-    (datetime.date(2019, 9, 21), _('Day 2')),
-    (datetime.date(2019, 9, 22), _('Day 3')),
+    (datetime.date(2019, 9, 5), _('Day 1')),
+    (datetime.date(2019, 9, 6), _('Day 2')),
```

Comment:
Oops

Reply:
Looks like this typo has been fixed in master. : )